lib/platform-api/requests.js

```diff
@@ -163,7 +167,11 @@ const _holdingsForBibs = async (bibs) => {
         return null
       }
     }))
-    return holdings.flat()
+
+    return holdings
+      .flat()
+      .map(holding => holding === null ? null : new SierraHolding(holding))
+      .filter(holding => !(holding instanceof SierraHolding && holding.getSuppressionWithRationale().suppressed))
```

Comment:
To avoid these ternaries, we should probably just filter out `null`s since there's no legit reason the `holdings` array should include a `null`

Reply:
Fixed